finalPrep/modZeroHero.js

```diff
@@ -1,41 +1,70 @@
 // Challenge - See if you can follow the instructions and complete the exercise in under 30 minutes!
 
 // Declare two variables - heroName AND specialAbility - set to strings
-
+var heroName = "Syntax Syd";
+var specialAbility = "resolve syntax errors";
 // Declare two variables - greeting AND catchphrase
+var greeting = ("Hello! I am " + heroName + "!");
+var catchphrase = `"Stop experiencing terrors, I am here to " ${specialAbility}`;
+
 //   greeting should be assigned to a string that uses concatenation to include the heroName
 //   catchphrase should be assigned to a string that uses interpolation to include the specialAbility
 
 // Declare two variables - power AND energy - set to integers
+var power = 3500;
+var energy = 7323;
+
+var fullPower = (power * 500);
+var fullEnergy = (energy + 150);
 
 // Declare two variables - fullPower AND fullEnergy
 //   fullPower should multiply your current power by 500
 //   fullEnergy should add 150 to your current energy
 
 // Declare two variables - isHuman and identityConcealed - assigned to booleans
-
+var isHuman = false;
+var identityConcealed = true;
 
 // Declare two variables - archEnemies AND sidekicks
 //   archEnemies should be an array of at least 3 different enemy strings
 //   sidekicks should be an array of at least 3 different sidekick strings
 
-// Print the first sidekick to your console
+var archEnemy = ["tiredness", "anxiety", "lima beans"];
+var sidekicks = ["comma", "semicolon", "brackets"]
 
+// Print the first sidekick to your console
+console.log(sidekicks[0]);
 // Print the last archEnemy to the console
-
+console.log(archEnemy[2])
 // Write some code to add a new archEnemy to the archEnemies array
-
+archEnemy.push("not reading error messages");
 // Print the archEnemies array to console to ensure you added a new archEnemy
-
+console.log(archEnemy);
 // Remove the first sidekick from the sidekicks array
+sidekicks.shift();
+sidekicks.push("quotes");
 
 // Print the sidekicks array to console to ensure you added a new sidekick
+console.log(sidekicks);
 
 // Create a function called assessSituation that takes three arguments - dangerLevel, saveTheDay, badExcuse
 //   - dangerLevel should be an integer
-//   - saveTheDay should be a string a hero would say once they save the day 
+//   - saveTheDay should be a string a hero would say once they save the day
 //   - badExcuse should be a string a hero would say if they are too afraid of the dangerLevel
 
+function assessSituation(dangerLevel, saveTheDay, badExcuse) {
```

Comment:
Logic looks great!

Style wise - you'll want to bring your `else if` and `else` lines up next to the previous closing `}`